### PR TITLE
Security/explicitly deny exportation

### PR DESCRIPTION
**Pull request by ken5scal**
# What
It seems all of the activities are private activities that do not expect receiving intents from other apps. Therefore it is safer to explicitly declare the activity as non-exportable activity in AndroidManifest.xml.
This was mentioned in [Android secure coding guide published by JPCert](https://www.jssec.org/dl/android_securecoding.pdf).

# Question
Of course the thing is... as long as you don't specify intent-filters, then exported attribute is set as "false" by default. So I was not too sure if this is necessary or not.

### CODE REVIEW

**Comment by konifar** [comment]
:eyes:

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="io.github.droidkaigi.confsched">
+<manifest
```
Comment (hotchemi) [inline app/src/main/AndroidManifest.xml]: Why this change is needed?

**Comment by hotchemi** [comment]
I agree with @ken5scal . Yup, the default value is false, but we're able to avoid unexpected security issue by declare exported attribute explicitly.

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (ken5scal) [inline app/src/main/AndroidManifest.xml]: woops, unintentional change. I will make a fix

**Comment by konifar** [comment]
Thanks! I don't know this. Looks good.

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="io.github.droidkaigi.confsched">
+          package="io.github.droidkaigi.confsched">
```
Comment (ken5scal) [inline app/src/main/AndroidManifest.xml]: God, my formatter is stupid....

**Comment by ken5scal** [comment]
Solved formatting issue!

**Comment by konifar** [comment]
LGTM
